tests/test_data/test_sim_data.py

```diff
@@ -180,8 +180,8 @@ def test_to_json():
     FNAME = "tests/tmp/sim_data_refactor.json"
     DATA_FILE = "tests/tmp/sim_extra_data.hdf5"
     sim_data.to_file(fname=FNAME)
-    # sim_data2 = SimulationData.from_file(fname=FNAME)
-    # assert sim_data == sim_data2
+    sim_data2 = SimulationData.from_file(fname=FNAME)
```

Comment:
this test will not pass because the `.json` no longer stores all the data. it is simply ignored and a special string tag is written in its place. Do we want to try to support writing data to separate file again? otherwise I think we can just keep this commented out.